pyvrp/search/tests/test_LocalSearch.py

```diff
@@ -21,20 +21,13 @@
 from pyvrp.tests.helpers import make_heterogeneous, read
 
 
-def test_local_search_returns_same_solution_when_there_are_no_operators():
```

Comment:
This test duplicates a test further below. I noticed that while writing the same docstring twice :)